### DISCUSSION TITLE
Support Cancel unstaking

### DISCUSSION THREAD

**Discussion by rubgithub** [akash-network/community#488] [Akash Delegation Framework]
Add Support do cancel staking process

![image](https://github.com/akash-network/community/assets/3399476/da6c4994-52fb-470f-84ce-10310e2f89a7)

**Comment by rodri-r** [comment]
Hey, this is already supported in every wallet, staking app

**Reply by rubgithub** [reply]
Nope, I'm using keplr and akash there's no cancel option.
$ATOM you can cancel

**Reply by rodri-r** [reply]
Hey, sorry I thought your meant before actually having tokens being unstaked.  I don't think Akash has such an option or any wallet

**Comment by anilmurty** [comment]
Thanks for this suggestion. While we (core team and community) haven't heard many requests for this - I can personally see the value. This would be potentially a good task for someone in the community to take on(?) by writing a more detailed proposal
